install.sh

```diff
@@ -21,7 +21,7 @@ EARLY_PACKAGES="gcc10 brotli c_ares libcyrussasl libidn2 libmetalink libnghttp2
 libtirpc libunistring openldap rtmpdump zstd ncurses ca_certificates libyaml ruby libffi \
 openssl nettle krb5 p11kit libtasn1 gnutls curl git icu4c"
 
-LATE_PACKAGES="crew_profile_base less most manpages filecmd mawk readline perl pcre pcre2 python27 python3 \
+LATE_PACKAGES="crew_profile_base less most manpages filecmd mawk readline perl pcre pcre2 python2 python3 \
```

Comment:
If you insist that `depends_on 'py3_pip'` shouldn't be a dependency of python3 then we should add `py3_pip` here instead.